plugins/refetch-interval/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@pinia/colada-plugin-refetch-interval",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "0.0.1",
+  "description": "Refetch queries on a regular interval with Pinia Colada",
+  "author": {
```

Comment:
Feel free to add yourself if you want: https://docs.npmjs.com/cli/v11/configuring-npm/package-json#people-fields-author-contributors